intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -7,7 +7,7 @@
 Introduction
 ------------
 PyTorch 1.8 includes an updated profiler API capable of
-recording the CPU side operations as well as the CUDA kernel launches on the GPU side.
+recording the CPU side operations as well as the CUDA kernel launches on the GPU side (``AMD ROCm™`` GPUs are not supported).
```

Comment:
@jeffdaily is this indeed the case?

Reply:
Also, why are you adding a backtricks here?

Reply:
ROCm has full upstream support of both modern pytorch profiling via kineto + ROCm's libroctracer, as well as the older autograd profiler via ROCm's roctx.

Run any application, pytorch or otherwise, using ROCm's rocprof.  The PyTorch GPU traces will be collected as part of your rocprof output.

Reply:
@malfet I quoted this function [`supported_activities`](https://github.com/pytorch/pytorch/blob/7f1cbc8b5a7de8794c36833baa47bb1343833589/torch/profiler/profiler.py#L35) in official PyTorch API which does not mention anything about ROCm GPUs. I suppose they should have something here for the ROCm profiling if that's the case.

Also the backtricks is due to an error in PySpelling test where `ROCm` is not in dictionary, so the test caught it as a misspelled word. I suppose I can either do this or add it in a custom dictionary list.

Reply:
We should update that comment.  ROCm profiler uses roctracer library to trace on-device HIP kernels.  Passing `ProfilerActivity.CUDA` is the correct activity to use for both CUDA and ROCm due to ROCm PyTorch's strategy of "masquerading" as CUDA so that users do not have to make any changes to their pytorch models when running on ROCm.  PyTorch chose to expose "CUDA" in public APIs and we chose to reuse them to make the transition to ROCm from CUDA easier on users.

Reply:
See https://pytorch.org/docs/stable/notes/hip.html.